README.md

```diff
@@ -47,3 +47,17 @@ I’m comfortable with git, the Linux environment, ES6, webpack, SASS. This past
  - [Linkedin](https://linkedin.com/in/pkshreeman)
  - [Stackoverflow](https://stackoverflow.com/story/pkshreeman)
  - [Github](https://github.com/pkshreeman)
+
+ ### Issues
+
+Issue: Travis CI build failure
+
+Solution (by Miguel T Rivera):
+
+1. Analyzed the build log and noticed it could not recognize ES6 syntax.
+2. Decided it was probably old and that files needed to be updated.
+3. Noticed some files were updated recently and others were 2 years old.
+4. Tried updating the files using git but ran into issues.
+5. Manually replaced each old file with the recent version.
```

Comment:
Also, Can you please pull request on Draft? I would like to be able to clone it on draft in general before we merge it into master.  You can create your branch here if you want to and makes you feel more comfortable, and just push that into repo so we can see what changes we are doing, and merge it?

Reply:
I'll probably close this PR and submit a new one. Also, can put in the readme what the different branches purposes? I wasn't sure which one to submit the PR.